app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -229,10 +229,10 @@ def idv_phone_otp_submitted(phone_number:, success:, failure_reason: nil)
 
     # The user reached the rate limit for Idv phone OTP submitted
     # @param [String] phone
```

Comment:
can you change "phone" to "phone_number"? other than that LGTM.

Reply:
Nice!  Good catch - thanks!